### DISCUSSION TITLE
关于notebook中运行时某些情况是否可以添加输入框的建议

### DISCUSSION THREAD

**Discussion by LXiaoMiao** [PaddlePaddle/PaddleNLP#688] [Q&A] [answered]
- 问题背景：对于paddlenlp未内置的数据集需从千言数据集等下载，解压后存入./data目录
- 问题来源：如果notebook重启并重新运行下载该数据集。会出现类似“文件夹已存在，是否替换“的提示信息，但是无法进行选择。具体如下图：
- ![L` _Y{DU4W@(UCU3SUYJ5SB](https://user-images.githubusercontent.com/57809596/123666050-73c01300-d86b-11eb-80e1-7e9866605a98.png)
- 解决建议：是否可以像调用input时出现输入框。具体如下图:
- ![BIZMI%8V6IJQKY{S@)0 27J](https://user-images.githubusercontent.com/57809596/123666928-3c9e3180-d86c-11eb-94c0-4448de83f561.png)

**Comment by libertatis** [comment, marked as answer]
你把解压的命令改一下：
```
!unzip -o ./SE-ABSA16_CAME.zip -d ./data
```
`-o` 覆盖已存在的文件且不要求用户确认。
